modules/ROOT/pages/manage-api-health-checks.adoc

```diff
@@ -0,0 +1,37 @@
+= Managing API Health Checks
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API health checks enable you to check connection attempts to a specific endpoint. When you configure the health check, specify a specific status code that indicates a healthy connection to the endpoint. If the endpoint connection attempt gives a status that violates the set parameters, an email address you specify receives an email noting the failed health check.
+
+You can use any 1xx, 2xx, 3xx, 4xx, or 5xx status codes for your health checks. The default health check states that the status code must equal 200, which is the OK status. If you expect successful connections to your endpoints, 2xx status codes are ideal connection results.
+
+For example, when the American Flights API is live, the status code should return 200, indicating the API can receive connections. When you configure a health check with these parameters, you receive an email when the endpoint has a status other than 200.
+
```

Comment:
```suggestion

Health checks have three states that appear in the *API Administration* page next to an API instance:

Add::
Indicates the API instance does not have a health check attached to it

Pass::
Indicates the health check returns the HTTP status code that you designated as healthy

Fail:: 
Indicates the health check returns an HTTP status code other than the code you designated as healthy

```

Reply:
Suggestion: Instead of "designated as healthy" ---> "configured"